backend/apps/volontulo/views/api.py

```diff
@@ -13,6 +13,7 @@
 from django.shortcuts import get_object_or_404
 from django.utils.encoding import force_bytes, force_text
 from django.utils.http import urlsafe_base64_encode, urlsafe_base64_decode
+from django_filters.rest_framework import DjangoFilterBackend
```

Comment:
I think there should be introduced i.e. isort and linting to check and fix imports for us to avoid this kind of discussions in the future.

BTW, this order is correct, isn't it (django_filters > django)? ;)

Reply:
IMHO it is in alphabetic order.

Reply:
yes, my bad here!